diff --git a/node_modules/@ai-sdk-tools/memory/dist/providers/redis.cjs b/node_modules/@ai-sdk-tools/memory/dist/providers/redis.cjs
index 87f5f9b..c9a1084 100644
--- a/node_modules/@ai-sdk-tools/memory/dist/providers/redis.cjs
+++ b/node_modules/@ai-sdk-tools/memory/dist/providers/redis.cjs
@@ -25,7 +25,7 @@ __export(redis_exports, {
 module.exports = __toCommonJS(redis_exports);
 
 // ../debug/dist/index.js
-var isDebugEnabled = process.env.DEBUG_AGENTS === "true";
+var isDebugEnabled = false;
 var colors = {
   reset: "\x1B[0m",
   gray: "\x1B[90m",
diff --git a/node_modules/@ai-sdk-tools/memory/dist/providers/redis.js b/node_modules/@ai-sdk-tools/memory/dist/providers/redis.js
index 790365d..241d4fa 100644
--- a/node_modules/@ai-sdk-tools/memory/dist/providers/redis.js
+++ b/node_modules/@ai-sdk-tools/memory/dist/providers/redis.js
@@ -1,5 +1,5 @@
 // ../debug/dist/index.js
-var isDebugEnabled = process.env.DEBUG_AGENTS === "true";
+var isDebugEnabled = false;
 var colors = {
   reset: "\x1B[0m",
   gray: "\x1B[90m",
diff --git a/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.cjs b/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.cjs
index 224b3ba..7cdd3c3 100644
--- a/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.cjs
+++ b/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.cjs
@@ -25,7 +25,7 @@ __export(upstash_exports, {
 module.exports = __toCommonJS(upstash_exports);
 
 // ../debug/dist/index.js
-var isDebugEnabled = process.env.DEBUG_AGENTS === "true";
+var isDebugEnabled = false;
 var colors = {
   reset: "\x1B[0m",
   gray: "\x1B[90m",
diff --git a/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.js b/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.js
index 6cba92d..1e5fa14 100644
--- a/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.js
+++ b/node_modules/@ai-sdk-tools/memory/dist/providers/upstash.js
@@ -1,5 +1,5 @@
 // ../debug/dist/index.js
-var isDebugEnabled = process.env.DEBUG_AGENTS === "true";
+var isDebugEnabled = false;
 var colors = {
   reset: "\x1B[0m",
   gray: "\x1B[90m",
