diff --git a/node_modules/react-router/dist/development/chunk-JMJ3UQ3L.mjs b/node_modules/react-router/dist/development/chunk-JMJ3UQ3L.mjs
index 67db8da..fc31146 100644
--- a/node_modules/react-router/dist/development/chunk-JMJ3UQ3L.mjs
+++ b/node_modules/react-router/dist/development/chunk-JMJ3UQ3L.mjs
@@ -373,7 +373,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/development/chunk-SZQUWNVJ.js b/node_modules/react-router/dist/development/chunk-SZQUWNVJ.js
index 8e0565c..96ef3fd 100644
--- a/node_modules/react-router/dist/development/chunk-SZQUWNVJ.js
+++ b/node_modules/react-router/dist/development/chunk-SZQUWNVJ.js
@@ -373,7 +373,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/development/index-react-server.js b/node_modules/react-router/dist/development/index-react-server.js
index c519b43..b945a52 100644
--- a/node_modules/react-router/dist/development/index-react-server.js
+++ b/node_modules/react-router/dist/development/index-react-server.js
@@ -298,7 +298,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/development/index-react-server.mjs b/node_modules/react-router/dist/development/index-react-server.mjs
index 91fc639..87274d8 100644
--- a/node_modules/react-router/dist/development/index-react-server.mjs
+++ b/node_modules/react-router/dist/development/index-react-server.mjs
@@ -277,7 +277,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/production/chunk-2HFJAX7U.js b/node_modules/react-router/dist/production/chunk-2HFJAX7U.js
index 91ae88c..19af700 100644
--- a/node_modules/react-router/dist/production/chunk-2HFJAX7U.js
+++ b/node_modules/react-router/dist/production/chunk-2HFJAX7U.js
@@ -373,7 +373,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/production/chunk-TINMVEA2.mjs b/node_modules/react-router/dist/production/chunk-TINMVEA2.mjs
index 7b3f2b8..ddadb0f 100644
--- a/node_modules/react-router/dist/production/chunk-TINMVEA2.mjs
+++ b/node_modules/react-router/dist/production/chunk-TINMVEA2.mjs
@@ -373,7 +373,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/production/index-react-server.js b/node_modules/react-router/dist/production/index-react-server.js
index c519b43..b945a52 100644
--- a/node_modules/react-router/dist/production/index-react-server.js
+++ b/node_modules/react-router/dist/production/index-react-server.js
@@ -298,7 +298,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
diff --git a/node_modules/react-router/dist/production/index-react-server.mjs b/node_modules/react-router/dist/production/index-react-server.mjs
index 91fc639..87274d8 100644
--- a/node_modules/react-router/dist/production/index-react-server.mjs
+++ b/node_modules/react-router/dist/production/index-react-server.mjs
@@ -277,7 +277,7 @@ var RouterContextProvider = class {
    */
   constructor(init) {
     __privateAdd(this, _map, /* @__PURE__ */ new Map());
-    if (init) {
+    if (init && Symbol.iterator in Object(init)) {
       for (let [context, value] of init) {
         this.set(context, value);
       }
